variants/postgis.Dockerfile.template

```diff
@@ -21,3 +21,5 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	./configure && \
 	make && \
 	make install
+
+RUN echo "CREATE EXTENSION postgis;" > /docker-entrypoint-initdb.d/postgis.sql
```

Comment:
I would suggest moving this into the `RUN` statement above to avoid creating another layer (unless there is a specific reason to not do this?). Otherwise it looks good :)